src/Servers/Kestrel/Core/test/Microsoft.AspNetCore.Server.Kestrel.Core.Tests.csproj

```diff
@@ -19,6 +19,7 @@
     <Compile Include="$(SharedSourceRoot)test\Shared.Tests\runtime\Http2\*.cs" LinkBase="Shared\runtime\Http2" />
     <Compile Include="$(SharedSourceRoot)test\Shared.Tests\runtime\Http3\*.cs" LinkBase="Shared\runtime\Http3" />
     <Compile Include="$(SharedSourceRoot)TestResources.cs" LinkBase="Shared" />
+    <Compile Include="$(SharedSourceRoot)test\Shared.Tests\PathNormalizerTests.cs" LinkBase="/" />
```

Comment:
I think this is causing these tests to run again. They should already be running from the Shared.Tests.csproj.

Reply:
Yes, I can remove this line.

Reply:
Updated. I wonder though how "discoverability" can be improved as the shared files are linked ie. in Kestrel.slnf but changing and finding corresponding unit tests are not that easy as the projects are not linked. Or is there a better slnf to edit.

ie. I did not notice this file exists until the failure in the CI, while I knew about the PathNormaizers.